.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: Flutter CI
+name: Master branch CI
```

Comment:
Probably worth changing this to "Flutter samples" as it is the string that shows up in the test status button in the README.md. Which also means the link needs to be changed, as the URL is going to change.